src/main/java/autoconfigure/KeycloakSecurityConfig.java

```diff
@@ -52,6 +51,7 @@
 
 @ConditionalOnProperty(prefix = "keycloak", name = "enabled", havingValue = "true", matchIfMissing = true)
 @KeycloakConfiguration
+@Import({ KeycloakConfigResolverConfig.class })
 public class KeycloakSecurityConfig extends KeycloakWebSecurityConfigurerAdapter {
```

Comment:
The circular dependency injection is in the parent class.

![image](https://user-images.githubusercontent.com/78334846/148345324-7717169b-1fd9-4761-ab7a-141ed117ed68.png)